cli/cmd/tapRunner.go

```diff
@@ -261,23 +253,26 @@ func updateMizuTappers(ctx context.Context, kubernetesProvider *kubernetes.Provi
 	return nil
 }
 
-func cleanUpMizu(kubernetesProvider *kubernetes.Provider) {
-	telemetry.ReportAPICalls(config.Config.Tap.GuiPort)
-	cleanUpMizuResources(kubernetesProvider)
-}
-
-func cleanUpMizuResources(kubernetesProvider *kubernetes.Provider) {
+func finishMizuException(kubernetesProvider *kubernetes.Provider) {
```

Comment:
why finishMizuException? it is not necessarily means it throws exception from what I understand (t will just be called in the end of the function because we used 	defer finishMizuException(kubernetesProvider) (line 103)